mozilla/blurts-server
# Change `premium` in URL to `plus`

# References: 
Jira: MNTOR-2896
Figma: N/A




# Description

Redirects from `/welcome-to-premium` to `/welcome-to-plus`.

# Screenshot (if applicable)

![image](https://github.com/mozilla/blurts-server/assets/4251/45fd8e2c-eb57-4cdb-b982-8f9ebc349060)

# How to test

Create a new user in the US, perform the scan, then upgrade to Plus. While at SubPlat, make sure to give the use Plus in the dev admin page. Then when you get sent back to `/welcome-to-premium`, it should redirect you to `/welcome-to-plus`. Then in the future, we can change the URL directly at SubPlat.

# Checklist (Definition of Done)
